src/index.js

```diff
@@ -90,7 +64,7 @@ class ReactCreditCards extends React.Component {
       nextNumber += '•';
     }
 
-    if (['amex', 'dinersclub'].includes(this.issuer)) {
```

Comment:
Maybe we should continue to use the old names?
Simple abstractions that can save users a lot of pain.

Or even better we could use a map to handle several common patterns? `amex`, `americanexpress`, `american-express` as this is used for preview too

Reply:
I created the map, makes sense to have an extensible way of handling these issues